tests/io/v3/vm/test_files.py

```diff
@@ -0,0 +1,33 @@
+"""
+Testing the File endpoint
+"""
+import os
+
+import responses
+
+FILE_BASE_URL = r"https://cloud.tenable.com/api/v3/file"
+BASE_URL = r"https://cloud.tenable.com"
+
+
+@responses.activate
+def test_upload(api):
+    '''
+    Test the file upload endpoint
+    '''
+    responses.add(
+        responses.POST,
+        f"{FILE_BASE_URL}/upload",
+        json={
+            "fileuploaded": "scan_targets_test.txt"
+        }
+    )
+
+    dummy_file_path = os.path.join(
+        os.path.dirname(
+            os.path.abspath(__file__)
+            ), "scan_targets_test.txt"
+        )
+    fobj = open(dummy_file_path)
```

Comment:
Hi @dhvanilshah-tenable have u tested if a file is not present at this given location than does it create one or throw an exception?

Reply:
@sahils-tenable If the file is not present on Tenable, it will create one. As for the tests, I have committed the file in our directory itself.

Reply:
okay cool!